tests/test_load_nifti.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import os
+import numpy as np
+import nibabel as nib
+from parameterized import parameterized
+from monai.transforms.transforms import LoadNifti
+
+TEST_CASE_1 = [
+    {
+        'as_closest_canonical': False,
+        'image_only': True
+    },
+    'test_image.nii.gz',
+    (128, 128, 128)
+]
+
+TEST_CASE_2 = [
```

Comment:
NAME these TEST_IMAGE_ONLY (or as you like) instead of 1,2. Makes it more readable what this case is for.

Reply:
Updated in the latest patch.
Thanks.